src/new/wallet.js

```diff
@@ -807,72 +770,55 @@ class HathorWallet extends EventEmitter {
    *
    * @param {UtxoOptions} options Utxo filtering options
    *
-   * @return {UtxoDetails} Utxos and meta information about it
+   * @return {Promise<UtxoDetails>} Utxos and meta information about it
    *
    */
-  getUtxos(options = {}) {
-    storage.setStore(this.store);
-    const historyTransactions = Object.values(this.getFullHistory());
+  async getUtxos(options = {}) {
+    const newOptions = {
+      token: options.token,
+      authorities: 0,
```

Comment:
This is a new options parameter, right? We should add to the docstring

Reply:
Done